### ISSUE TITLE
Responsively on Mac forcing updates which break the app and prevent usage

### ISSUE THREAD

**Issue by johnstonesnow** [responsively-org/responsively-app#802]
I am running Mac OSX 10.14.6. 
I am testing a site I am building, and responsively told me to update. I clicked to do that, and it broke it, it won't run now, says I am running osx 10.14.6 but this app 'requires osx 10.13.x'. 
thankfully I have timemachine backups so I restored yesterday's version of the app. But now when I run it, it does open and run but there's a message saying 'Update has been downloaded and will install automatically upon exit' or words to that effect. 
Obviously I don't want that. How do I stop it?
Thanks

**Comment by manojVivek** [comment]
That's strange. Do you have a screenshot of the error it throws? That states the osx version?

**Comment by johnstonesnow** [comment]
Hi, 

Sure, screenshot attached. 

I ran the program, instantly came up and now won't run without me reverting to older version. I need to find the downloaded update if possible to stop it constantly updating automatically when I close it. Can you help me do that? I can roll back to older version which ran fine (I stupidly clicked to agree to update even though the update should know it can't run on my machine!), but whilst it sees that it has downloaded the update, it's going to keep update after each time I run it. 

<a href="https://imgbb.com/"><img src="https://i.ibb.co/hgKsqGx/R.png" alt="R" border="0"></a>

**Comment by manojVivek** [comment]
Yeah, unfortunately there is no straightforward way to disable the auto-update right now. You can look at blocking the network request to the GitHub releases as a workaround but that would be challenging as well.

**Comment by manojVivek** [comment]
Also, to get some clarity on the problem, are you using an Intel Mac or an Apple Silicon Mac?  

We have two separate builds for these two machine architectures, and guessing if this is caused by running a wrong binary from https://responsively.app/download/.

<img width="622" alt="Screenshot 2023-03-01 at 7 32 10 PM" src="https://user-images.githubusercontent.com/1283424/222162742-f5595cd4-0868-43fb-8ac8-e011dc55469c.png">

**Comment by johnstonesnow** [comment]
Maybe I didn't explain well enough. It worked perfectly for a year or so, so I am pretty sure I must have the correct version. I think this is just a case (I often get) where software updates in line with OS updates, but I lag behind (deliberately as I detest Apple and their telemetry, which increases with each update). I think Responsively is trying to force me to update because it's 'assuming' I (like almost everybody else) now have a newer version of OSX or a Silicon machine (I have a 2013 iMac). I don't! 

The best suggestion is blocking the network, if I had done that before it would have prevented this but now I think it's too late as the software has a download of newer version already downloaded and the app is sure it needs to update every time it runs (hence I think blocking connections won't make any difference). I run Little Snitch (partly to block Apple's horrible spyware) so I can easily block all outbound connections from Responsively. I am sure it won't work for reasons already explained but will give it a try. thanks

**Comment by manojVivek** [comment]
I understand your concern with not upgrading the OS, I'll see if I can explicitly set the minimum required version somehow.

**Comment by johnstonesnow** [comment]
Thank you, and it's refreshing to hear someone who understands my concerns regarding the OS updates! I am usually met with "just do it"! I fear Linux may have to be explored shortly, just about had enough now! Thanks again, I can use it in the mean time anyway so don't go to great trouble if I am the only one with this problem :)
